### PR TITLE
fixing NodePort example

### PR DESCRIPTION
**Pull request by yuvalif**
fixing issue #57

### CODE REVIEW

File: workloads/virtual-machines/expose-service.md [hunk 1]
```diff
@@ -147,4 +184,4 @@ NAME      TYPE           CLUSTER-IP       EXTERNAL-IP                   PORT(S)
 lbsvc     LoadBalancer   172.30.27.5      172.29.10.235,172.29.10.235   27017:31829/TCP   5s
 ```
 
-Use `vinagre` client to connect your VirtualMachine by using the public IP and port.
+Use `vinagre` client to connect your VirtualMachine by using the public IP (172.29.10.235) and port (31829) that were dynamically allocated fo the service.
```
Comment (stu-gott) [inline workloads/virtual-machines/expose-service.md]: s/fo/of/

**Comment by yuvalif** [comment]
@stu-gott  and @vladikr  any point in merging that? too many conflicts accumulated. would recommend on cancelling.

**Comment by stu-gott** [comment]
@yuvalif This PR was simply fixing examples right? I assume what you mean is that the examples were fixed in a different way during the OVM->VM rename?

**Comment by yuvalif** [comment]
@stu-gott , yes, the samples were already fixed and modified by other PRs.

**Pull request closed without merging**